design/FY2019/Interzone Air Wall NFP and Design.md

```diff
@@ -0,0 +1,192 @@
+that the results are reasonable. Various levels of reporing including design component loads will all need to
+be consistent with each other.
+
+## Input Output Reference Documentation ##
+
+The proposed input mechanism is to modify the `BuildingSurface:Detailed` and `FenestrationSurface:Detailed` objects to allow a new
+choice of *AirWall* for Surface Type. When this is selected, the Construction input is left blank (or ignored with a warning).
+ Also, the only allowed choices for Outside Boundary Condition will be *Surface* and *Zone*.
+
+## Input Description ##
+
+**BuildingSurface:Detailed**
+
+Field: Surface Type
+
+. . . *AirWall* indicates an open boundary between two zones. When this option is selected, Construction Name is ignored and the
+Outside Boundary Condition must be either *Surface* or *Zone*.
+
+Field: Construction Name
+
+. . . This field is ignored if the Surface Type is *AirWall*.
+
+Field: Outside Boundary Condition
+
+. . . If Surface Type is *AirWall* the only valid options are *Surface* and *Zone*.
+
+**FenestrationSurface:Detailed**
+
+Field: Surface Type
+
+. . . *AirWall* indicates an open boundary between two zones. When this option is selected, Construction Name is ignored and the
+Outside Boundary Condition must be either *Surface* or *Zone*.
+
+Field: Construction Name
+
+. . . This field is ignored if the Surface Type is *AirWall*.
+
+
+## Outputs Description ##
+
+Various output descriptions may need notes to explain what happens when energy crosses an airwall boundary.
+
+## Engineering Reference ##
+
+TBD
+
+## Example File and Transition Changes ##
+
+One or more example files will be developed which use airwalls in various ways.
+
+## References ##
+
+None.
+
+## Design ##
+
+The overall plan is to tackle radiant exchange first, then solar/daylighting, then air exchange.
+
+### Surface Class ###
+Anywhere that `Surface(n).Class` is used will be evaluated to add a case or if block for 
+the new `SurfaceClass_AirWall`.
+
+### SuperZones ###
+For the superzone concept, a second zone number field will be added to the surface data
+structure(s). For solar, daylighting, and radiant exchange, the alternate superzone number will
+be used to determine which surfaces to use and how they interact.
+
+Will this work? Probably not initially. There are places that assume all of the surfaces in a
+zone are adjacent to each other in the `Surface` data structure. e.g.
+
+```
+        for (ISurf = Zone(ZoneNum).SurfaceFirst; ISurf <= Zone(ZoneNum).SurfaceLast; ++ISurf) {
+```
+
+This example is from daylighting, so it will need to be changed. Other examples of this usage occur
+in convection coefficients and other places where the current method will not be impacted. 
+This can be addressed by using zone-level lists of surface numbers (which is already being 
+developed in a separate effort).
+
+### Reporting ###
+Various zone-level outputs will be reviewed to determine the proper accounting of 
```

Comment:
What are the cases that trigger warnings or errors?  Can you have AirWalls with no adjacent surface?  Can it be self-adjacent?  What about zone multipliers?  Can you have subsurfaces on an AirWall?  Zone mixing or airflow network in addition to the airwall?

Reply:
Lots of good questions that I'll need to flesh out in the NFP. Some intitial clarifications:
1. The AirWall must have an adjacent surface.
2. The AirWall cannot be self-adjacent. I suppose it *could* be allowed but that wouldn't accomplish anything - it would have zero impact on the simulation. The whole idea is to have an opening that lets two zones transmit solar and radiant between them.
3. The intent is that the superzones would be determined simply by looking at the airwall surfaces which must all be an interzone surface. The zone which holds the airwall and the zone which holds it's adjacent surface would be grouped together.
4. Zone multipliers? Good question. The only way this could work would be if all of the zones grouped into the superzone had the same multiplier.
5. The airwall as proposed wouldn't do any mixing itself, so yes, zone mixing and/or airflow network are required to move air between the zones.

Reply:
6. No subsurfaces on an AirWall.
7. Perhaps an AirWall subsurface should be handled differently than an AirWall base surface to prevent the entire building from being connected into a single superzone. Maybe the new choice for fenestration objects (subsurfaces) should be "Opening" and it would behave as an interzone window with perfect transmittance. Something to consider.

Reply:
Yeah, this superzone concept just makes me think the user should model the whole space as one zone instead...allow interfaces to deal with the space diversity issues that come with grouping the individual zones into one larger zone before handing it off to the energy calculation engine.

But alas.

If we are going to have an air wall, you've definitely covered a lot, laid out a lot of pitfalls, and clearly have a handle on how complex this will be.  I would say that, since we seem to have ramped up end-user-usage of the airflow network, special emphasis should be given there to ensure this works well with that.  I look forward to seeing this evolve into a full design.

Reply:
The whole point is that the HVAC equipment and internal gains are different between the zones, so combining zones in the first place is not doable.